src/app/components/weather-alert/weather-alert.component.html

```diff
@@ -6,6 +6,7 @@
     </div>
     <div class="toast-body">
       <p>{{weatherAlert?.activeAlertDescription}}</p>
+      <div class="text-end"><a class="text-decoration-none bg-primary ps-1 pe-1 rounded text-light h6 bg-opacity-70"routerLink="/weatherAlert"><i>View More...</i></a></div>
```

Comment:
Text-decorarion-none may be all that's needed.  This is an anchor tag styled as a button.  This being button drive more attention to the button rather than the message.